ledger/participant-state/kvutils/src/main/scala/com/daml/ledger/participant/state/kvutils/KeyValueCommitting.scala

```diff
@@ -34,10 +34,14 @@ import scala.collection.JavaConverters._
 class KeyValueCommitting private[daml] (
     engine: Engine,
     metrics: Metrics,
-    inStaticTimeMode: Boolean) {
+    inStaticTimeMode: Boolean,
+    packageValidation: CommitPackageValidationMode,
+) {
   private val logger = LoggerFactory.getLogger(this.getClass)
 
-  def this(engine: Engine, metrics: Metrics) = this(engine, metrics, false)
+  // RH: what is the good default value for packageValidation
```

Comment:
I'd suggest removing the comment and leaving this into `Postcommit` mode and we'll resolve this later.